api/src/test/java/org/pmiops/workbench/db/dao/InstitutionDaoTest.java

```diff
@@ -106,4 +112,55 @@ public void test_uniqueDisplayNameRequired() {
         new DbInstitution().setShortName("Inst2").setDisplayName("Not Unique");
     institutionDao.save(snowflake2);
   }
+
+  @Test
+  public void testGetReportingInstitutions() {
+    DbInstitution institution = ReportingTestUtils.createDbInstitution();
+    institutionDao.deleteAll();
+    institution = institutionDao.save(institution);
+
+    final List<ProjectedReportingInstitution> institutions =
+        institutionDao.getReportingInstitutions();
+    assertThat(institutions).hasSize(1);
+    ReportingTestUtils.assertInstitutionFields(institutions.get(0));
+  }
+
+  @Test
+  public void testGetReportingInstitutions_empty() {
+    institutionDao.deleteAll();
+    assertThat(institutionDao.getReportingInstitutions()).isEmpty();
+  }
+
+  @Test
+  public void testGetReportingInstitutions_multiple() {
+    institutionDao.deleteAll();
+
+    DbInstitution institution1 = ReportingTestUtils.createDbInstitution();
+    institution1.setDisplayName("Cairo Consulting");
+    institution1.setDuaTypeEnum(DuaType.RESTRICTED);
+    institution1.setOrganizationTypeEnum(OrganizationType.OTHER);
+    institution1.setOrganizationTypeOtherText("Pyramid Scheme");
+
+    DbInstitution institution2 = new DbInstitution();
+    institution2.setShortName("mash");
+    institution2.setDisplayName("MASH");
+    institution2.setDuaTypeEnum(DuaType.MASTER);
+    institution2.setOrganizationTypeEnum(OrganizationType.HEALTH_CENTER_NON_PROFIT);
+
+    institutionDao.save(ImmutableList.of(institution1, institution2));
+
+    final List<ProjectedReportingInstitution> institutions =
+        institutionDao.getReportingInstitutions();
+    assertThat(institutions).hasSize(2);
+    assertThat(institutions.get(0).getDisplayName()).isEqualTo("Cairo Consulting");
+    assertThat(institutions.get(0).getDuaTypeEnum()).isEqualTo(DuaType.RESTRICTED);
+    assertThat(institutions.get(0).getOrganizationTypeEnum()).isEqualTo(OrganizationType.OTHER);
+    assertThat(institutions.get(0).getOrganizationTypeOtherText()).isEqualTo("Pyramid Scheme");
+
+    assertThat(institutions.get(1).getDisplayName()).isEqualTo("MASH");
+    assertThat(institutions.get(1).getDuaTypeEnum()).isEqualTo(DuaType.MASTER);
+    assertThat(institutions.get(1).getOrganizationTypeEnum())
+        .isEqualTo(OrganizationType.HEALTH_CENTER_NON_PROFIT);
+    assertThat(institutions.get(1).getDisplayName()).isEqualTo("MASH");
+  }
```

Comment:
does `getOrganizationTypeOtherText()` return NULL or `''` for inst 2?  Add an assert for the correct one

Reply:
Projections don't do any conversion at all to what's in the DB, so you get whatever is in the table. If the application enforces nullity, that's great, but I don't think we need tests to show that all nullable columns can be null.  

If we're going to tackle null vs empty, I'd prefer to have a standalone story for that to do it app-wide, maybe even in an interceptor 😈 .

Reply:
ok